.github/workflows/run.yml

```diff
@@ -31,4 +31,4 @@
             REDISCLOUD_URL: ${{ secrets.REDISCLOUD_URL }}
             GPP_URL: ${{ secrets.GPP_URL }}
             GPP_KEY: ${{ secrets.GPP_KEY }}
-          run: python scheduler/scripts/run_greedymax.py
+          run: python scheduler/scripts/run_greedymax.py ERROR
```

Comment:
Even if these changes don't make so much improve in run time it would be nice to manage these more dynamically for the whole system (both the script and the service itself). Could you make it that the level is set from a env variable ? that could allows us to modify the levels without deploying. Probably through or config.yml file.

Reply:
Sure, I will!